lib/game/player/easy_computer_player.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Game.Player.EasyComputerPlayer do
+  @moduledoc """
+  The module that handles an Easy Computer Players move
+  """
+
+  alias Game.Player.PlayerBehaviour
+  @behaviour PlayerBehaviour
+
+  def valid_input(board, communicator) do
```

Comment:
Consider adding `@impl` see here: https://elixir-lang.org/getting-started/typespecs-and-behaviours.html#adopting-behaviours

Reply:
Forgot about that, got it, thank you! 😄